scalafmt-cli/src/main/scala/org/scalafmt/cli/ScalafmtCoreRunner.scala

```diff
@@ -67,6 +70,7 @@ object ScalafmtCoreRunner extends ScalafmtRunner {
     catch {
       case MisformattedFile(_, diff) =>
         options.common.err.println(diff)
+        if (options.check) Breaks.break
```

Comment:
`scalafmt --check` on first failure will breaks here before `ExitCode.TestError` returned to the `exitCode` variable. Then how will `--check` will return exit code 1?

Reply:
hmmm... let me see.

Reply:
moved this check.